aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx

```diff
@@ -0,0 +1,82 @@
+'use client'
```

Comment:
- remove this use client, it unnecessary

Reply:
NextJS 13 and new versions use useParams from next/navigation which works only on client side